README.md

```diff
@@ -1 +1,212 @@
+  - [💻 link to Kanban board information ](#-link-to-kanban-board-information-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+# 📖 Final Capstone Front End <a name="about-project"></a>
+
+**Final Capstone Front End** The front-end component of this final capstone project is developed using React.js, designed to provide an intuitive user interface for managing creation of places to rent and make reservations for those places. It communicates with the back-end API built with Ruby on Rails.
+
+## 💻 Link to Back End <a name="link-to-back-end"></a>
+
+[Link to Back End](https://github.com/grauJavier/final-capstone-back-end)
+
+<p align="right"\><a href="#readme-top"\>back to top\</a></p>
+
+## 💻 link to Kanban board information <a name="built-with"></a>
+
+[Link to Kanban Board](https://github.com/grauJavier/final-capstone-back-end/projects/1)
+
+[Initial state](https://github.com/grauJavier/final-capstone-front-end/issues/38)
+
+This project was completed by three Team members:
+
+- Javier Grau
+- Manuel Sanchez
+- Anthony Vásquez
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **React.js Components**
+- **API Communication with Backend**
+- **User-Friendly Interface**
+- **Responsive Design**
+- **State Management**
+
+<p align="right"\><a href="#readme-top"\>back to top\</a></p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://rent-eaze.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Before you begin, make sure you have the following prerequisites installed on your system:
+
+- Node.js: You need Node.js to run the React.js application.
+- npm: npm is used to manage packages in your React project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+sh <br>
+cd my-folder <br>
+git clone git@github.com:grauJavier/final-capstone-front-end.git
+
+### Install
+
+Install this project with:
+
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start
```

Comment:
- [ ] This command won't work:

![image](https://github.com/grauJavier/final-capstone-front-end/assets/57421823/74f228e0-f4e6-4ae4-8cfd-ea1386b27de4)

The correct command to run the frontend project is `npm run dev`.  Kindly correct it.

Reply:
Hello!
Thanks for your feedback 🙌🏼 
This is solved 😄